plugins/backstage-plugin-flux/README.md

```diff
@@ -277,6 +278,70 @@ kubernetes:
           caData: LS0tLS1CRUdJTiBDRVJUSUZJQ0...
 ```
 
+6. [Optional] Add a Card to a Page
+
+- Create a page: An example of a basic page can be found at [Flux Runtime Page](https://github.com/weaveworks/weaveworks-backstage/plugins/backstage-plugin-flux/src/components/FluxRuntimePage/index.tsx/).
+
+- Add the page to the plugin exports:
+
+```tsx
+// In plugins/backstage-plugin-flux/src/plugin.ts
+
+/**
+ * Page used to show Flux Controllers / Deployments in Flux Runtime
+ * @public
+ */
+export const FluxRuntimePage = weaveworksFluxPlugin.provide(
+  createRoutableExtension({
+    name: 'FluxRuntimePage',
+    component: () =>
+      import('./components/FluxRuntimePage').then(m => m.FluxRuntimePage),
+    mountPoint: rootRouteRef,
+  }),
+);
+```
```

Comment:
Don't think we need this bit

Reply:
I mean, I think we have a copy of this code in the plugin that user can import and use. If that works